docs/redo-log-archiving.md

```diff
@@ -0,0 +1,116 @@
+
+You can verify if MySQL has configured the archive directory with the following command:
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT @@innodb_redo_log_archive_dirs;
+```
+
+The result either provides the path or returns NULL, the archiving directory is not set.
+
+```{.bash data-prompt"}
+$ mkdir -p /var/lib/mysql-redo-archive/backup1
+$ chown mysql. -R /var/lib/mysql-redo-archive
+$ chmod _R 700 /var/lib/mysql-redo-archive/
+```
+
+The `innodb_redo_log_archive_dirs` global variable contains labeled directories. The format is `label:directory-path`. Multiple objects can be listed with a semi-colon between each set.
+
+```{.bash data-prompt="mysql>"}
+mysql> SET PERSIST innodb_redo_log_archive_dirs='backup1:/var/lib/mysql-redo-archive/';
+```
+
+Verify that the archive was created with the following command:
+
+```{.bash data-prompt="mysql>"}
+mysql> SHOW GLOBAL VARIABLES LIKE 'innodb_redo_log_ar%';
+```
+
+??? example "Expected output"
+    ```{.text .no-copy}
+    +------------------------------+--------------------------------------+
+    | Variable_name                | Value                                |
+    +------------------------------+--------------------------------------+
+    | innodb_redo_log_archive_dirs | backup1:/var/lib/mysql-redo-archive/ |
+    +------------------------------+--------------------------------------+
+    1 row in set (0.0025 sec)
+    ```
+
+Users with the [INNODB_REDO_LOG_ARCHIVE] privilege can call the `innodb_redo_log_archive_start()` or `innodb_redo_log_archive_stop()` functions. to either start or stop archiving. 
+
+```{.bash data-prompt="mysql>"}
+mysql> SELECT innodb_redo_log_archive_start('backup1','backup1');
+```
+
+??? example "Expected output"
+    ```{.text .no-copy}
+    +-----------------------------------------------------+
+    | innodb_redo_log_archive_start('backup1','backup1')  |
+    +------------------------------+----------------------+
+    |                                                   0 |
+    +------------------- ---------------------------------+
+    1 row in set (0.00225 sec)
+    ```
+
+## Percona XtraBackup and redo log archiving
+
+Percona XtraBackup 8.0.34-29 added the [redo-log-arch-dir](xtrabackup-option-reference.md#-redo-log-arch-dirname) option. If the archiving directory is not configured in MySQL, Percona XtraBackup creates a temporary directory for the archive file. To use this option, you must run Percona XtraBackup as the owner of `mysqld`. If Percona XtraBackup does not have the appropriate permissions, then the action generates a "Permission denied" error, and archiving is not used.
+
+```{.bash data-prompt}
+$ sudo -H -u mysql bash
+$ xtrabackup --no-lock=1 --compress --parallel=4 --host=localhost --user=root --password='password_string' --backup=1 --target-dir=/Backup/13oct 2> /tmp/b0-with-redo-archiving-as-mysql-os-user.log
```

Comment:
Where is the `--redo-log-arch-dir` parameter been set?

Reply:
I don't understand the question.